src/components/actividades/ListadoActividades.js

```diff
@@ -85,12 +87,27 @@ export default function ListadoActividades() {
 
   const validar = anyPass([validarNombreActividad, validarNombreEspacio]);
 
-  const actividadesFiltradas = useMemo(() => filter(validar, actividades), [
-    actividades,
-    validar,
-  ]);
+  const actividadesFiltradas = useMemo(
+    () => compose(filter(validar))(actividades),
```

Comment:
Si hay una sola función, no tiene sentido componer:


```suggestion
    () => filter(validar, actividades),
```